functions/src/project/delete-project.ts

```diff
@@ -29,6 +30,8 @@ async function deleteProjectRepositories(projectUid: string, project: ProjectMod
               }
 
               t.update(repoRef, { projects: repoData.projects });
+            
+              deleteRepoBuilds(repositoryUid);
```

Comment:
I dont think this is still part of the transaction https://firebase.google.com/docs/firestore/manage-data/transactions 

I will look into it

Reply:
This is being done as part of https://github.com/DashboardHub/PipelineDashboard/pull/1604